bin/MadGraph5_aMCatNLO/cards/production/13TeV/Singlet_Triplet_Higgs_portal_DM_ms200_mt220_ct3mm/Singlet_Triplet_Higgs_portal_DM_proc_card.dat

```diff
@@ -0,0 +1,7 @@
+import model Singlet_Triplet_Higgs_portal_DM
+define l+ = e+ mu+ ta+
+define l- = e- mu- ta-
+define vl = ve vm vt
+define vl~ = ve~ vm~ vt~
+generate p p > psiplus psiminus , ( psiplus > w+ psil, w+ > l+ vl), ( psiminus > w- psil, w- > l- vl~) @0
```

Comment:
There is no available phase space with this syntax.
m(psiplus) = 220 < m(w+) + m(psil) = 80 + 200 = 280
With your syntax, psiplus, w+, and psil are all forced to be onshell which does not make sense physically.
You must modify it to a syntax like "psiplus > l+ vl psil".

Again as expected, when I run with your cards it gives an error.
Zero result detected:  No Phase Space. Please check particle masses.

Reply:
Indeed you're correct. For some reason, I didn't observe this error while I tried to produce events locally. I will make a change to the proc card accordingly and make a fresh commit.